projects/angular/src/forms/common/label.ts

```diff
@@ -36,6 +49,16 @@ export class ClrLabel implements OnInit, OnDestroy {
     return this.el.nativeElement && this.el.nativeElement.textContent;
   }
 
+  /**
+   * Allowing signposts inside labels to work without disabling default behavior. <label> is spreading a click event to its children so signposts get
+   * automatically closed once clicked inside a <label>.
+   * @param event
+   */
+  @HostListener('click', ['$event'])
+  onClick(event) {
```

Comment:
```suggestion
  private onClick(event) {
```

This can be marked `private` so that it's not part of the public api.